05/index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles/reset.css">
+    <link rel="stylesheet" href="./styles/style.css">
+    <link rel="stylesheet" href="./styles/mobile.css">
+    <link rel="stylesheet" media="(min-width: 800px)" href="./styles/desktop.css">
+
+
+    <title>Korona Gór Polski</title>
+</head>
+
+<body>
+    <header class="header">
+        <nav class="header__nav">
+            <h1 class="header__logo">Korona Gór Polski</h1>
+            <label for="menu"></label>
+            <input type="checkbox" id="menu">
+            <ul class="nav__list">
+                <li class="nav__item">Klub</li>
+                <li class="nav__item">Historia</li>
+                <li class="nav__item">Szczyty</li>
+                <li class="nav__item">Aktualności</li>
+                <li class="nav__item">Kontakt</li>
+            </ul>
+        </nav>
+    </header>
+    
+    <div id="flexbox">
+    <section class="content__one">
```

Comment:
Powiedziedziałbym, że to nie jest `element` a `block` czyli `content`.
Jeśli potrzebujesz rozróżnić tą sekcję od następnej to zrobiłbym modyfikator `content content--one`.
Natomiast jeśli "upierasz" się na elemencie tj. `content__one` to brakuje w przodku (nie koniecznie rodzicu) zaznaczenia blocku tj. `content`. Aby istniał element musi być najpierw w strukturze block.